model.js

```diff
@@ -0,0 +1,117 @@
+const fs = require('fs')
+
+class Model {  
+    constructor() {
+        this.data = this.readData()
+    }
```

Comment:
constructor pada model seharusnya berisi properti dari setiap data, misalnya: this.id, this.task, this.status, dll.

Reply:
function" pada model juga seharusnya adalah static function